src/gui/search/searchsortmodel.h

```diff
@@ -49,8 +52,45 @@ class SearchSortModel: public QSortFilterProxyModel
 
     explicit SearchSortModel(QObject *parent = 0);
 
+    void enableNameFilter(bool enabled);
+    void setNameFilter(const QString& searchTerm = QString());
+
+    //! \brief setSizeFilter Sets parameters for filtering by size
```

Comment:
Too bad Qt Creator doesn't do this. I am a bit skeptical of this. It might lead to loss of readability. But anyway, don't change this. This isn't a big problem for me ATM.